cmd/rancher_release/set_kdm_branch_references.go

```diff
@@ -67,22 +66,104 @@ func setKDMBranchReferencesCommand() *cli.Command {
 }
 
 func setKDMBranchReferences(c *cli.Context) error {
+	var err error
 	forkPath := c.String("fork-path")
+	if forkPath == "" {
+		forkPath, err = cmdPath()
+		if err != nil {
+			return err
+		}
+		logrus.Info("fork path: " + forkPath)
+		if err := isGitRepo(forkPath); err != nil {
+			return err
+		}
+	}
 	baseBranch := c.String("base-branch")
-	currentKDMBranch := c.String("current-kdm-branch")
+	if baseBranch == "" {
+		baseBranch, err = currentBranch(forkPath)
+		if err != nil {
+			return err
+		}
+		logrus.Info("base branch: " + baseBranch)
+	}
 	newKDMBranch := c.String("new-kdm-branch")
-	createPR := c.Bool("create-pr")
+	logrus.Info("new KDM Branch: " + newKDMBranch)
 	forkOwner := c.String("fork-owner")
 	githubToken := c.String("github-token")
 	dryRun := c.Bool("dry-run")
+	logrus.Info("dry run: " + strconv.FormatBool(dryRun))
+	createPR := c.Bool("create-pr")
+	logrus.Info("create PR: " + strconv.FormatBool(createPR))
 	if createPR {
 		if forkOwner == "" {
-			return errors.New("'create-pr' requires 'fork-owner'")
+			if forkOwner, err = gitRepoOwner(forkPath); err != nil {
+				return err
+			}
+			logrus.Info("fork owner: ", forkOwner)
 		}
 		if githubToken == "" {
 			return errors.New("'create-pr' requires the 'GITHUB_TOKEN' env var")
 		}
 	}
+	return rancher.SetKDMBranchReferences(context.Background(), forkPath, baseBranch, newKDMBranch, forkOwner, githubToken, createPR, dryRun)
+}
+
+func cmdPath() (string, error) {
+	path, err := os.Getwd()
```

Comment:
I'm not sure this needs to be another function. There's nothing happening in here that wouldn't otherwise.